payment_twikey/models/account_move.py

```diff
@@ -273,19 +276,20 @@ def _compute_twikey_url(self):
         Calculate the url of the invoice in Twikey.
         """
         try:
-            twikey_client = (self.env["ir.config_parameter"].sudo().get_twikey_client(company=self.env.company))
-            for move in self:
-                if move.twikey_invoice_identifier:
-                    move.twikey_url = twikey_client.invoice.geturl(move.twikey_invoice_identifier)
+            for move in self.filtered("twikey_invoice_identifier"):
+                twikey_client = self.env["ir.config_parameter"].sudo().get_twikey_client(company=move.company_id)
```

Comment:
Het zou logischer zijn om `get_twikey_client()` op `res.company` te definiëren ipv `ir.config_parameter`, maar die refactor gaat wellicht buiten de scope van dit issue.

Reply:
Ja, je ziet op het ticket nog een hele lijst van zaken die verbeterd zouden moeten worden, maar we moeten het voor nu even beperken.